product-imges.go

```diff
@@ -36,6 +36,16 @@ func (dtos ProductImageDTOs) ToShopify() shopify.ProductImages {
 	return productImages
 }
 
+func buildProductImageDTOs(productImages shopify.ProductImages) ProductImageDTOs {
```

Comment:
this should be an exported function like the other dto builders

Reply:
Ah the `dto` builder I looked at in `line-items` which is probably the only one not exported :sweat_smile:

Reply:
oooh would you mind fixing that one?